src/helpers/schema.sql

```diff
@@ -19,6 +19,7 @@ CREATE TABLE proposals (
   ipfs VARCHAR(64) NOT NULL,
   author VARCHAR(64) NOT NULL,
   created INT(11) NOT NULL,
+  updated INT(11) DEFAULT NULL,
```

Comment:
Maybe also add an `INDEX` for this field ?

Reply:
not sure. Do you think we need this index?

Reply:
There's an option to use on WHERE query such as updated ==, !=, >, and <, so I think it'll benefit from an index